app/views/students/show.html.erb

```diff
@@ -15,7 +15,11 @@
       <div class="card">
         <div class="card-body">
           <div class="d-flex flex-column align-items-center text-center">
-            <img src="<%= @student.avatar_src %>" width="150"/>
+            <% if @student.avatar %>
+              <img src="<%= @student.avatar_src %>" width="150"/>
```

Comment:
maybe we could use image_tag here as well

Reply:
I tested it but it didn't work 🤔

Reply:
never mind, I tested it wrong 😅

Reply:
😂